### ISSUE TITLE
how do you use multiple connections.

### ISSUE THREAD

**Issue by kenan0620** [AndyConlisk/Signal-R-ObjectiveC-Presentation#1]
this example is for one connections.how do you use if you use multiple connections .thanks.

**Comment by AndyConlisk** [comment]
I never had a need to use Multiple connections.
I used this library to have iOS connect to signalR - https://github.com/DyKnow/SignalR-ObjC
I would look there for information on multiple connections.

**Comment by kenan0620** [comment]
you




柯南
On 09/14/2017 22:09, Andy Conlisk wrote:

I never had a need to use Multiple connections.
I used this library to have iOS connect to signalR - https://github.com/DyKnow/SignalR-ObjC
I would look there for information on multiple connections.

**Comment by kenan0620** [comment]
You use more connections in one project?




柯南
在2017年09月14日 22:10，柯南 写道：
you




柯南
On 09/14/2017 22:09, Andy Conlisk wrote:

I never had a need to use Multiple connections.
I used this library to have iOS connect to signalR - https://github.com/DyKnow/SignalR-ObjC
I would look there for information on multiple connections.

**Comment by kenan0620** [comment]
Can you speak Chinese?




柯南
在2017年09月14日 22:11，柯南 写道：
You use more connections in one project?




柯南
在2017年09月14日 22:10，柯南 写道：
you




柯南
On 09/14/2017 22:09, Andy Conlisk wrote:

I never had a need to use Multiple connections.
I used this library to have iOS connect to signalR - https://github.com/DyKnow/SignalR-ObjC
I would look there for information on multiple connections.

**Comment by AndyConlisk** [comment]
No I am sorry, I can not speak Chinese.

I am not sure what you mean by multiple connections.
This project was for a presentation I did years ago to show it was possible to connect an iOS app to a SignalR connection. 
I used the library I linked previously to do the connecting from iOS to SignalR. 
You would have a better chance at an answer to your question there. https://github.com/DyKnow/SignalR-ObjC

**Comment by kenan0620** [comment]
Well, thank you very much for your answer. I now use signalR in my project. I means: I create a SRHubConnection for one project. This connection then creates multiple proxies for data delivery.My project use the MVVM framework. So let's think about that.





At 2017-09-14 22:16:35, "Andy Conlisk" <notifications@github.com> wrote:


No I am sorry, I can not speak Chinese.

I am not sure what you mean by multiple connections.
This project was for a presentation I did years ago to show it was possible to connect an iOS app to a SignalR connection.
I used the library I linked previously to do the connecting from iOS to SignalR.
You would have a better chance at an answer to your question there. https://github.com/DyKnow/SignalR-ObjC
